src/PeoplePage.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import { Person } from './types';
+import { getPeople } from './api';
+import { Loader } from './components/Loader';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
nice improvements but you need to change initial state of isLoading to true to fix problem with appearing "There are no people on the server"